maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -335,12 +340,34 @@ private void installOWAs(Server server, DistroProperties distroProperties) throw
     
     private void downloadOWAs(File targetDirectory, DistroProperties distroProperties, File owasDir) throws MojoExecutionException {
         List<Artifact> owas = distroProperties.getOwaArtifacts(distroHelper, targetDirectory);
-        OpenmrsBintray openmrsBintray = new OpenmrsBintray(getProxyFromSettings());
-
         if (!owas.isEmpty()) {
             wizard.showMessage("Downloading OWAs...\n");
             for (Artifact owa: owas) {
-                openmrsBintray.downloadOWA(owasDir, owa.getArtifactId(), owa.getVersion());
+                wizard.showMessage("Downloading OWA: " + owa);
+                installOwa(owasDir, owa);
+            }
+        }
+    }
+
+    private void installOwa(File owasDir, Artifact owa) throws MojoExecutionException {
+        // Support existing Bintray artifacts
+        try {
+            OpenmrsBintray openmrsBintray = new OpenmrsBintray(getProxyFromSettings());
+            openmrsBintray.downloadOWA(owasDir, owa.getArtifactId(), owa.getVersion());
+        }
+        catch (Exception e) {
+            // If file is not found in bintray, and exception will be thrown.  Try to find it in Maven repo.
+            moduleInstaller.installModule(owa, owasDir.getAbsolutePath());
```

Comment:
Tangential, but I guess we will want to change the name of this from "moduleInstaller" to "artifactInstaller" at some point? (but agree, let's not do it now)

Reply:
Yes, agreed, but didn't want to overcomplicate this PR.  I think the entire SDK could do with a "3.0" version that makes significant updates without worry for backwards compatibility.